src/Directory.Build.props

```diff
@@ -12,22 +12,19 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Akka Persistence journal and snapshot store backed by MongoDB database.</Description>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <LangVersion>latest</LangVersion>
   </PropertyGroup>
   <PropertyGroup>
     <NetCoreTestVersion>netcoreapp3.1</NetCoreTestVersion>
     <NetTestVersion>net6.0</NetTestVersion>
     <NetFrameworkTestVersion>net471</NetFrameworkTestVersion>
     <NetStandardLibVersion>netstandard2.0</NetStandardLibVersion>
 
+    <AkkaVersion>1.5.8</AkkaVersion>
```

Comment:
I have seen some projects have two variables for Akka and Akka Hosting. Not sure if required
https://github.com/akkadotnet/Akka.Persistence.Sql/blob/8052a17c75a678ca8052923a3d8ee43a19fb2bd7/src/Directory.Build.props#L24C1-L24C53